src/Bank.js

```diff
@@ -0,0 +1,51 @@
+// Dependencies
+
+const Card = require("./Card.js");
+const PrintStatement = require("./printStatement.js");
+
+class Bank {
+  cards = [];
```

Comment:
id' expect to see these proeprties in the constructor because you have a constructor